src/main/kotlin/no/nav/familie/ef/mottak/service/SakService.kt

```diff
@@ -38,15 +38,16 @@ class SakService(private val integrasjonerClient: IntegrasjonerClient,
         val fagsakOpprettet = journalposter.any { it.sak?.fagsaksystem == INFOTRYGD && it.sak?.fagsakId != null }
```

Comment:
Gjelder egentlig ikke denne PR-en, men ser vi går igjennom alle journalposter for ENF - men er det ikke spesifisert om saken/søknaden gjelder overgangsstønad, barnetrygd eller skolepenger?

Reply:
Det er en veldig god observasjon, som jeg tenkte på og glemte for en uke siden. Det må fikses hvis ikke så må vi lete ettter feilen når vi begynner med banretilsyn og skolepenger.